src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
-<ui-table [elements]="elements" (selected)="onSelected($event)"></ui-table>
-<ui-detail-panel [element]="selected" *ngIf="selected"></ui-detail-panel>
\ No newline at end of file
+<app-ui-table [elements]="elements" (selected)="onSelected($event)"></app-ui-table>
```

Comment:
Followed the Angular semantic configured in `angular.json` as it suggest to put `app` prefix on selector